src/main/java/myGames/Utils.java

```diff
@@ -68,4 +68,20 @@ public static int oneOrZero() throws IOException {
             return oneOrZero();
         }
     }
+    public static String chooseItem() {
+        try {
+            String item = READER.readLine();
+            if (item.equals("Stone") || item.equals("Paper") || item.equals("Scissors")){
```

Comment:
For Stone/Paper/Scissors strings we can create Enum, because a lot of repetitions

Reply:
Ok, I will do it!